src/java.desktop/share/classes/javax/sound/SoundClip.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package javax.sound;
+
+import java.io.File;
+import java.io.IOException;
+
+import com.sun.media.sound.JavaSoundAudioClip;
+
+/**
+ * The {@code SoundClip} class is a simple abstraction for playing a sound clip.
+ * It will play any format that is recognized by the {@code javax.sound} API,
+ * and for which it has support. This includes midi data.
+ * Playing sound requires that the environment grants access to audio devices.
+ * Typically this means running the application in a desktop environment.
+ * <p>
+ * Multiple {@code SoundClip} items can be playing at the same time, and
+ * the resulting sound is mixed together to produce a composite.
+ *
+ * @since 25
+ */
+public final class SoundClip {
+
+    private final JavaSoundAudioClip clip;
+
+    /**
+     * Creates a {@code SoundClip} instance which will play a clip from the supplied file.
+     * <p>
+     * If the file does not contain recognizable and supported sound data, or
+     * if the implementation does not find a suitable output device for the data,
+     * playing the clip will be a no-op.
+     *
+     * @param file the file from which to obtain the sound data
+     * @return a {@code SoundClip}
+     * @throws IllegalArgumentException if {@code file} is {@code null}
+     * @throws IOException if there is an error reading from {@code file}
+     */
+    public static SoundClip createSoundClip(File file) throws IOException {
```

Comment:
Can we gain any benefits by using Path instead?

Reply:
or maybe inject all these simple methods into `javax.sound.sampled.Clip` or its subclass.